js/implementation.native.ts

```diff
@@ -44,13 +44,13 @@ let updatesEnabled = false;
  */
 export function setRNConfiguration(config: GeolocationConfiguration) {
   RNCGeolocation.setConfiguration({
-    ...config,
+    skipPermissionRequests: config.skipPermissionRequests,
```

Comment:
`skip...` is now optional in `config`, so needs to be explicit here to remain consistent.